src/main/flight/pid_init.c

```diff
@@ -47,9 +47,9 @@
 
 #if defined(USE_D_MIN)
 #define D_MIN_RANGE_HZ 80    // Biquad lowpass input cutoff to peak D around propwash frequencies
-#define D_MIN_LOWPASS_HZ 10  // PT1 lowpass cutoff to smooth the boost effect
-#define D_MIN_GAIN_FACTOR 0.00005f
-#define D_MIN_SETPOINT_GAIN_FACTOR 0.00005f
+#define D_MIN_LOWPASS_HZ 30  // PT1 lowpass cutoff to smooth the boost effect
```

Comment:
Let's update the comments according to the changes:
```cpp
// PT2 lowpass input cutoff to peak D around propwash frequencies
// PT2 lowpass cutoff to smooth the boost effect
```

Reply:
Done, thanks!